### DISCUSSION TITLE
kernel compilation l4t compilation

### DISCUSSION THREAD

**Discussion by htex0189** [OE4T/meta-tegra#1148] [Q&A] [answered]
Hi, 

I have the following error while compiling the l4t-35 5.10.kernel. 

./include/linux/lockref.h:19:10: fatal error: generated/bounds.h: No such file or directory
   19 | #include <generated/bounds.h>

bounds.h or other files defined in the Kbuild are not created in the include/generated folder.

I compile the kernel using 

make ARCH=arm64 CROSS_COMPILE=aarch64-linux-gnu- -j8

Thank you,

**Comment by ichergui** [comment, marked as answer]
Hey @htex0189 

Could you please give us some context ? 
Did you enabled any specific Linux kernel token ? (I mean, did made changes to the Linux kernel configuration ?)

Building the [Linux kernel / Linux-tegra](https://github.com/OE4T/linux-tegra-5.10) is working fine.
Please follow the instructions described in our [wiki](https://github.com/OE4T/linux-tegra-5.10/wiki/Building-Linux-tegra-5.10.x-from-sources) page

**Comment by htex0189** [comment]
Hi,

Thanks for the wiki link. 
The kernel compilation is good.

Is there any documents detailing the procedure to install the compiled kernel and modules on to Jetson PC please?

**Reply by ichergui** [reply]
Hey @htex0189 
I will check and will share here.

**Reply by htex0189** [reply]
Hi @ichergui Thanks

**Reply by htex0189** [reply]
Hi @ichergui do you have some information for flashing the Jetson with compiled kernel please

**Reply by dwalkes** [reply]
@htex0189 there are some suggestions in https://github.com/OE4T/meta-tegra/discussions/997 which may be useful.
